.github/workflows/python-package.yml

```diff
@@ -54,3 +54,15 @@ jobs:
           TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
         shell: pwsh
         run: twine upload dist/*
+
+      - name: Setup edge
+        uses: browser-actions/setup-edge@latest
+        if: ${{ matrix.browsers == 'edge' }}
+
+      - name: Setup firefox
+        uses: browser-actions/setup-firefox@latest
+        if: ${{ matrix.browsers == 'firefox' }}
+
+      - name: Setup chrome
+        uses: browser-actions/setup-chrome@latest
+        if: ${{ matrix.browsers == 'chrome' }}
```

Comment:
The test settings for WebSurfer is here https://github.com/microsoft/autogen/blob/main/.github/workflows/contrib-tests.yml#L183-L218

Reply:
Thank you so much for your guidance with this @thinkall, very much appreciated! :)

Reply:
I moved the browser actions as we discussed.  The build tests are failing with the error [ModuleNotFoundError: No module named 'markdownify'](https://github.com/signalprime/autogen/blob/25e15e05cc95af0bad97c6cbc0324f4b3c259258/autogen/browser_utils.py#L6)

I confirmed `markdownify` is installed with the [`websurfer` requirements](https://github.com/signalprime/autogen/blob/25e15e05cc95af0bad97c6cbc0324f4b3c259258/setup.py#L56) [here](https://github.com/signalprime/autogen/blob/25e15e05cc95af0bad97c6cbc0324f4b3c259258/.github/workflows/contrib-tests.yml#L202) and [here](https://github.com/signalprime/autogen/blob/25e15e05cc95af0bad97c6cbc0324f4b3c259258/.github/workflows/contrib-openai.yml#L245), just before the browser actions are specified.  

I will research more to see how I can solve it.

Reply:
@afourney hi friend, if I'm not mistaken, you introduced the WebSurfer agent.  Could you offer any additional advice about how I could get this PR to trigger the websurfer extras installation during build tests and avoid import errors for libraries like `markdownify`?